Codeinwp/neve
# fix: mega menu not working on edge [closes #1347]

### Summary
Fixes issue with mega-menu `pointer-events: none` CSS property when mega menu is opened.


### Will affect visual aspect of the product

NO

### Test instructions


- Test on edge that the mega menu works as expected and doesn't close when you hover the root item.
- Tested on BrowserStack and everything works as expected


Closes #1347.
